artifacthub-repo.yml

```diff
@@ -0,0 +1,3 @@
+owners:
```

Comment:
@bornav @codefromthecrypt seems like this is a first step, with that we should be able to claim repository ownership and I could get repository ID

Reply:
here is the id of the one I created, if you can't transfer it to your account I would recommend creating a new one
071a8849-26f8-4d02-b8e2-898966d5a1a7